fails:Kernel#callcc is a private method
fails:Kernel#callcc is possible to exit a loop like a break
fails:Kernel#callcc is possible to call a continuation multiple times
fails:Kernel#callcc preserves changes to block-local scope
fails:Kernel#callcc preserves changes to method-local scope
fails:Kernel.callcc is possible to exit a loop like a break
fails:Kernel.callcc is possible to call a continuation multiple times
fails:Kernel.callcc preserves changes to block-local scope
fails:Kernel.callcc preserves changes to method-local scope
